### PR TITLE
Metadata server additions

### PR DESCRIPTION
**Pull request by robmyers**
Add metadata for:
* Contracts.
* Creature Accessories.

### CODE REVIEW

File: metadata-api/app.py [hunk 1]
```diff
@@ -32,12 +38,71 @@
...
+    'Message-in-a-bottle.png',
+    'Pearl.png',
+    'Scuba-mask.png',
+    'Trident.png'
+]
+ACCESSORIES_NAMES = [a.replace('-', ' ').replace('.png', '')
+                     for a in ACCESSORIES_IMAGES]
+ACCESSORIES_ATTS_INT = [200, 11, 3, 41, 9, 172]
+ACCESSORIES_ATTS_PERCENT = [5, 10, 1, 20, 15, 25]
+ACCESSORIES_ATTS_LOCATION = ['head', 'body', 'held', 'held', 'head', 'held']
```
Comment (dfinzer) [inline metadata-api/app.py:55]: Would you mind making these and the `BASES`, `EYES`, and `MOUTH` traits for the creatures capitalized? (i.e.,

```
['Head', 'Body', 'Held', 'Held', 'Head', 'Held']
```

We had a slight shift in strategy and are moving towards using human readable strings for traits (thanks for helping clean up)

File: metadata-api/app.py [hunk 2]
```diff
@@ -32,12 +38,71 @@
...
+
+CONTRACT_URI_METADATA = {
+    'opensea-creatures': {
+        'name': 'OpenSea Creatures',
+        'description': 'Friendly creatures of the sea.',
+        'image': 'https://example.com/image.png',
+        'external_link': 'https://openseacreatures.io'
+    },
+    'opensea-erc1155': {
+        'name': 'OpenSea ERC1155',
```
Comment (dfinzer) [inline metadata-api/app.py:85]: Can we call this `OpenSea Creature Accessories`?

File: metadata-api/app.py [hunk 3]
```diff
@@ -32,12 +38,71 @@
...
+        'description': "OpenSea's ERC1155 contract.",
```
Comment (dfinzer) [inline metadata-api/app.py:86]: `Fun and useful accessories for your OpenSea creatures.`

File: metadata-api/app.py [hunk 4]
```diff
@@ -32,12 +38,71 @@
...
+
+
+# contractURI() support
+
+CONTRACT_URI_METADATA = {
+    'opensea-creatures': {
+        'name': 'OpenSea Creatures',
+        'description': 'Friendly creatures of the sea.',
+        'image': 'https://example.com/image.png',
+        'external_link': 'https://openseacreatures.io'
```
Comment (dfinzer) [inline metadata-api/app.py:82]: I like your idea of linking to the github, so how bout `https://github.com/ProjectOpenSea/opensea-creatures/`

File: metadata-api/app.py [hunk 5]
```diff
@@ -54,57 +119,136 @@ def creature(token_id):
...
+    image_url = _compose_image(['images/box/lootbox.png'], token_id, 'box')
+
+    attributes = []
+    _add_attribute(attributes, 'number_inside', [3], token_id)
+
+    return jsonify({
+        'name': 'Accessory Loot Box',
+        'description': 'This lootbox contains some OpenSea Creature accessories! It can also be traded!',
+        'image': image_url,
+        'external_url': 'https://openseacreatures.io/accessory/%s' % token_id,
```
Comment (dfinzer) [inline metadata-api/app.py:227]: Nit: `accessory/box`?

File: metadata-api/app.py (hunk 1 above)
Comment (robmyers) [reply metadata-api/app.py:55]: Done!

File: metadata-api/app.py (hunk 2 above)
Comment (robmyers) [reply metadata-api/app.py:85]: Fixed.

File: metadata-api/app.py (hunk 3 above)
Comment (robmyers) [reply metadata-api/app.py:86]: Fixed.

File: metadata-api/app.py (hunk 4 above)
Comment (robmyers) [reply metadata-api/app.py:82]: Fixed.

File: metadata-api/app.py (hunk 5 above)
Comment (robmyers) [reply metadata-api/app.py:227]: I made it box/accessory to match the api url structure.

**Pull request merged**